Makefile

```diff
@@ -94,3 +95,10 @@ clean-all:
 clean:
 	./scripts/clean.sh
 
+# deploy services
+deploy-services:
+	./scripts/deploy-service.sh
+	
+# delete services
+delete-services:
+	./scripts/deploy-service.sh delete
```

Comment:
Just a nit, but we could probably change the name of the script to something like `manage-service.sh`.

Reply:
fixed